dspace/modules/xmlui-mirage2/src/main/webapp/themes/Mirage2/styles/_style.scss

```diff
@@ -58,4 +58,8 @@
   #aspect_submission_StepTransformer_item_publication-title{
     margin-bottom:5px;
   }
-}
\ No newline at end of file
+}
+
+body {
+    color: green;
```

Comment:
This file looks like an unintended inclusion from a local theme?

Reply:
Good catch, yes that was added accidentally with `git add -u`. Removed!